articles/architecture-scenarios/b2c.md

```diff
@@ -1,108 +1,144 @@
+[User profile management](/architecture-scenarios/implementation/b2c/user-profile-mgmt) will address the most common cases for the changes users will want to make to their profiles. We've found the most successful implementations address the following topics at this point during Phase 1, however precisely what you tackle will depend on your specific requirements (for example, you won't need to provide for user metadata management if you're not using user metadata):
 
-## GDPR support
+* Managing metadata
+* Resetting passwords
+* Verifying accounts
+* Blocking users
 
-If your application is likely to store data about users in the European Union, then your application is subject to the requirements of the [General Data Protection Regulation (GDPR)](/compliance/gdpr), which took effect on May 25th 2018. The GDPR adds some new requirements and significant new fines, so make sure your application complies with the regulations.
+### User Authorization
 
-Auth0 has [features to help you meet GDPR obligations](/compliance/gdpr/features-aiding-compliance). You can display a consent page and track the user’s consent via the Lock widget. The consent status can then be stored in the Auth0 user profile for each consenting user. With the Management API you can get users to satisfy data access requests, as well as give data to users in JSON format to satisfy data portability requirements.
+[User Authorization](/architecture-scenarios/implementation/b2c/user-authorization) is, for customers who have specific access control requirements, the next thing on the agenda, and the focus for Phase 1 will be centered on how custom ID Token Claims can be leveraged to support this.
 
-## Anomaly Detection
+### User logout
 
-An unfortunate part of modern life on the internet is hackers. Hackers are constantly trying to find a way into applications. For example, they may try to log in using common passwords. Or they may use credentials stolen from elsewhere, hoping that users re-used the same passwords at other sites.
+In the [user logout](/architecture-scenarios/implementation/b2c/user-logout) workstream, users will want to log out of your system and you'll need to decide exactly what this looks like. Auth0 supports several variations when it comes to user logout giving you flexibility to choose what works best for your implementation.
 
-Auth0's [Anomaly Detection](/anomaly-detection) detects these situations for Auth0 Database Connections and provides options for how to respond. Turn on Anomaly Detection and configure the response options so you can respond appropriately if such an event occurs.
+### Operations
 
-## Github Deployment 
+Operations can be addressed in parallel, though we'd recommend you setup your email provider early on as this will enable you to minimize disruption moving forward as well as allow you to quality test specific functionality not possible to do with out-of-the-box email provision. Topics to cover here will include:
```

Comment:
```suggestion
Operations can be addressed in parallel, though we'd recommend you setup your email provider early on as this will enable you to minimize disruption moving forward as well as allow you to quality test specific functionality not possible without-of-the-box email provision. Topics to cover here will include:
```